### PR TITLE
Installer script readme

### PR DESCRIPTION
**Pull request by rv602**
### What kind of change does this PR introduce?
I had recently made a [PR](https://github.com/PalisadoesFoundation/talawa-api/pull/1153) that added a Python script to automate the installation process of Talawa-API. It was merged a day back.
This PR adds the instructions to use it in the `INSTALLATION.md` file.

### Does this PR introduce a breaking change?
No

### Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?
Yes

### CODE REVIEW

File: src/types/generatedGraphQLTypes.ts [hunk 1]
```diff
@@ -2460,7 +2453,6 @@ export type Resolvers<ContextType = any> = {
   Time?: GraphQLScalarType;
   Translation?: TranslationResolvers<ContextType>;
   URL?: GraphQLScalarType;
-  UpdateUserTagInput?: UpdateUserTagInputResolvers<ContextType>;
   Upload?: GraphQLScalarType;
```
Comment (noman2002) [inline src/types/generatedGraphQLTypes.ts:2456]: Why we need these changes in this PR?

Comment (noman2002) [reply src/types/generatedGraphQLTypes.ts:2456]: In this whole file ?

Comment (rv602) [reply src/types/generatedGraphQLTypes.ts:2456]: @noman2002 I didn’t make that change.
It probably got changed when I made the commit : -


https://user-images.githubusercontent.com/92572013/225726297-2c545b1e-37d0-4559-89ba-fefda465c58c.mp4

<br>
Should I undo those changes and make a new commit?

Comment (noman2002) [reply src/types/generatedGraphQLTypes.ts:2456]: yes please

**Comment by noman2002** [comment]
@rv602 Any progress ?

**Comment by jordan-barrett-jm** [comment]
Is this ready to be merged?

**Comment by rv602** [comment]
There are some minor changes to be done. I will make the commit soon.

**Comment by rv602** [comment]
@noman2002 whenever a new commit is made, the `src/types/generatedGraphQLTypes.ts` is being modified as shown in the video in the above discussion : -

https://user-images.githubusercontent.com/92572013/227733076-9b83f359-fc46-4e29-8c6c-a460d79dbba3.mp4

Should i add the `src/types/generatedGraphQLTypes.ts` to the `.gitignore` file and make a new commit accordingly?

**Comment by noman2002** [comment]
@palisadoes This PR is open since 3 weeks, are we going to merge this? If not , should I close this ?

**Comment by palisadoes** [comment]
Closing

**Pull request closed without merging**